### DISCUSSION TITLE
Publish dataset

### DISCUSSION THREAD

**Discussion by hidehikoinagaki** [dandi/helpdesk#61] [Q&A] [answered]
Hi, sorry, most likely I am missing something really simple... I am trying to publish my dataset but nothing has happened after pressing the "Publish" button. Should I just wait for a while? Or is there anything else I need to do? My nwb files are validated and uploaded. Thank you!

**Comment by hidehikoinagaki** [comment]
It is now published. It will be great to have a message box or something saying it takes time etc

**Comment by waxlamp** [comment, marked as answer]
Hi @hidehikoinagaki! I apologize for the inconvenience. ~We are tracking this specific problem in https://github.com/dandi/dandi-archive/issues/760,~ and we realize that until we fix it, the user experience of publishing a dandiset is lacking.

Thank you for the feedback, and please continue to let us know where you're running into trouble!

**Reply by waxlamp** [reply]
A quick update: the issue I linked above is not the right one to address this problem. Instead, it's this: https://github.com/dandi/dandi-archive/issues/955.

Thanks again for reporting about your experience with the DANDI Archive!
